frontend/src/routes/FacilityBooking/CreateBooking/index.tsx

```diff
@@ -99,113 +114,25 @@ const RepeatWeeklyPickerRow = styled.div`
 export default function CreateBooking() {
   const dispatch = useDispatch()
   const history = useHistory()
-  const {
-    newBooking,
-    newBookingName,
-    newBookingFromDate,
-    newBookingToDate,
-    newBookingCCA,
-    newBookingDescription,
-    newBookingFacilityName,
-    facilityList,
-    isLoading,
-    ccaList,
-    createBookingError,
-    createSuccess,
-    createFailure,
-    newBookingFacilityId,
-    numRepeatWeekly,
-  } = useSelector((state: RootState) => state.facilityBooking)
+  const params = useParams<{ facilityId: string }>()
+  const { register, handleSubmit, errors, control } = useForm()
+  const [isWeeklyOn, setIsWeeklyOn] = useState(false)
```

Comment:
include the state's type
```suggestion
  const [isWeeklyOn, setIsWeeklyOn] = useState<boolean>(false)
```